src/ray/protobuf/gcs_service.proto

```diff
@@ -175,6 +175,12 @@ message GetAllNodeInfoRequest {}
 message GetAllNodeInfoReply {
   GcsStatus status = 1;
   repeated GcsNodeInfo node_info_list = 2;
+  GcsStats gcs_stats = 3;
+}
+
+message GcsStats {
+  repeated TaskSpec infeasible_tasks = 1;
```

Comment:
can you only report necessary fields?

Reply:
If you take a look at `test_actor_groups` in `test_actor.py`, these fields are necessary when gcs scheduling is enabled.